src/base/searchengine.cpp

```diff
@@ -383,6 +413,12 @@ void SearchEngine::updateNova()
         QFile::copy(":/" + novaFolder + "/nova2.py", filePath);
     }
 
+    filePath = searchDir.absoluteFilePath("nova2dl.py");
+    if (getPluginVersion(":/" + novaFolder + "/nova2dl.py") > getPluginVersion(filePath)) {
+        removePythonScriptIfExists(filePath);
```

Comment:
Since it is a kind of "revert" commit this probably does not matter, otherwise I would use here a helper to do the "rename,copy,remove,rename" sequence. In general, such helper might be of use in the project in general.